server/model/FlightPhoto.js

```diff
@@ -29,7 +29,7 @@ module.exports = (sequelize, DataTypes) => {
     size: {
       type: DataTypes.BIGINT,
     },
-    isYoutubeLink: {
+    isExternalLink: {
```

Comment:
Brauchen wir das oder ist das für den import der alten Daten?

Reply:
Habe es einfach mal drin gelassen, falls man in Zukunft auch auf externe Datenquellen verweisen möchte. Dann könnte man in path die externe Url speichern und über dieses Flag dann die Unterscheidung treffen.

Will halt mögliche Änderungen der Struktur in der DB in Zukunft vermeiden. Ist halt immer lästig, dass bei einer großen produktiven DB zu machen.